src/lineage/templates/clean_template.md

```diff
@@ -9,6 +9,36 @@ Upstream repository: [`{{ remote_url }}`]({{ remote_url }})
 Check the changes in this pull request to ensure they won't cause issues with
 your project.
 
+## ✅ Pre-approval checklist ##
+
+- Remove any of the following that do not apply.
+- If you're unsure about any of these, don't hesitate to ask. We're here to help!
+
+- [ ] Changes are limited to a single goal - *eschew scope creep!*
+- [ ] *All* future TODOs are captured in issues, which are referenced
+      in code comments.
+- [ ] All relevant type-of-change labels have been added.
+- [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
+- [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
+- [ ] All relevant repo and/or project documentation has been updated
+      to reflect the changes in this PR.
+- [ ] Tests have been added and/or modified to cover the changes in this PR.
+- [ ] All new and existing tests pass.
+
+## ✅ Pre-merge checklist ##
+
+- Remove any of the following that do not apply.
+- These boxes should remain unchecked until the pull request has been approved.
+
+- [ ] Bump version via the `bump_version.sh` script, if this
+      repository is versioned.
```

Comment:
I would like to clarify in this point that this should only be done if it is appropriate. Following semver not every group of Lineage changes should see a version change (GitHub Actions changes, pre-commit hook updates, etc.).

Reply:
Please see commit 3364e18.